packages/field/docs/CustomFieldsTutorial.md

```diff
@@ -0,0 +1,76 @@
+# Creating a custom field
```

Comment:
How about putting it at the end of the field readme?

Reply:
There is no Field readme yet;) Only the Form Fundaments, which kind of gives an overview of all form controls (fields and fieldsets).

Reply:
I created an issue to solve the 'relative link problem' for npm(https://github.com/ing-bank/lion/issues/13), so that we can keep separate files for documentation.